adapters/adform/adform_test.go

```diff
@@ -57,7 +58,7 @@ var adformTestData aBidInfo
 // Legacy auction tests
 
 func DummyAdformServer(w http.ResponseWriter, r *http.Request) {
```

Comment:
A minor suggestion: Since this is a helper function just for tests, it would be good to make it private so that it's not accessible outside this package.

Reply:
Noted for future, thanks! No capacity sadly to introduce it this time